provision-osx/tasks/configure_buildfarm_node.yml

```diff
@@ -1,118 +1,106 @@
 ---
 
--
-  debug:
-    msg:
-    - Configuring macOS nodes as part of the build farm will be skipped.
-    - credential_private_key_path, credential_public_key_path and credential_passphrase must be defined.
-    - Run this job again to configure the build farm nodes.
-  when:
-  - credential_private_key_path is not defined
-  - credential_private_key_path == ''
-  - credential_public_key_path is not defined
-  - credential_public_key_path == ''
-  - credential_passphrase is not defined
-  - credential_passphrase == ''
+- include: configure_node_env_vars.yml
 
 -
   block:
   -
-    name: Get jenkins url
-    shell: "oc get route jenkins --namespace={{ project_name }} -o jsonpath='{.spec.host}'"
-    delegate_to: "{{ groups['master'][0] }}"
-    register: route_output
-    changed_when: False
-
-  - include: configure_node_env_vars.yml
+    name: Generate SSH key pair for Jenkins
+    command: ssh-keygen -t rsa -f /Users/{{ ansible_ssh_user }}/.ssh/jenkins_id_rsa -N "{{ credential_passphrase | default('') }}"
+    args:
+      creates: "/Users/{{ ansible_ssh_user }}/.ssh/jenkins_id_rsa"
```

Comment:
Well the task will run but it will return ok as opposed to changed  status, command wont be executed